handlers/twiml/handlers.go

```diff
@@ -253,7 +253,16 @@ func (h *handler) Send(ctx context.Context, msg courier.MsgOut, res *courier.Sen
 
 		for _, comp := range msg.Templating().Components {
 			for varKey, varIndex := range comp.Variables {
-				contentVariables[varKey] = msg.Templating().Variables[varIndex].Value
+				value := msg.Templating().Variables[varIndex].Value
+				attType, value := handlers.SplitAttachment(value)
+				attType = strings.Split(attType, "/")[0]
+
+				// not attachment ignore the split, revert to original entire value
```

Comment:
@rowanseymour hold on I found a better way here

Reply:
checking the type `msg.Templating().Variables[varIndex].Type` so we can split as attachment